README.pod

```diff
@@ -98,40 +96,171 @@ C<dispose> method:
 
   $dbh.dispose;
 
-=head2 Fetching data
+=head2 Running Queries
+
+=head3 C<execute>
+
+For a single execution of a query you may use execute directly. This starts the query within the database and returns
+a C<StatementHandle> which will be used for [Retrieving Data](#retrieving-data).
+
+  $dbh.execute(q:to/SQL/);
+    CREATE TABLE tab (
+      id serial PRIMARY KEY
+      col text
+    );
+    SQL
+
+Errors occurring within the database for an SQL statement will typically throw an exception within Raku. The level of
+detail within the exception object depends on the database driver.
+
+  $dbh.execute('CREATE TABLE failtab ( id );');
+
+  CATCH {
+     when X::DBDish::DBError {
+        say .message;
+     }
+  }
+
+In order to build a query dynamically without risk of SQL injection you need to use parameter binding. The C<?>
+will be replaced by an escaped copy of the parameter provided after the query.
+
+  my $value-id = 19;
+  $dbh.execute('INSERT INTO tab (id) VALUES (?)', $value-id);
+
+  my $value-text = q{Complex text ' value " with quotes};
+  $dbh.execute('INSERT INTO tab (id, col) VALUES (?, ?)', $value-id, $value-text);
+
+  # Undefined or Nil values will be converted to NULL by parameter binding.
+  my $value-nil;
+  $dbh.execute('INSERT INTO tab (id, col) VALUES (?, ?), $value-id, $value-nil);
+
+
+Parameter binding should be used where-ever possible, even if the number of parameters is dynamic. In this case the
+number of elements for IN is dynamic. The number of C<?>'s is scaled to fit the list of items, and the list
+is provided to execute as individual items.
+
+  my @value-list = 1 .. 6;
+  my $parameter-bind-marks = @value-list.map({'?'}).join(',')  # ?,?,?,?,?,?
+  my $query = 'SELECT id FROM tab WHERE id IN (%s)'.sprintf($parameter-bind-marks);
+  $dbh.execute($query, |@value-list);
```

Comment:
Is the | a typo or some weird Raku construct?

Reply:
|@ is the opposite of a slurpy *@.

my @params = 1,2,3;
foo(|@params) is foo(1,2,3) rather than foo( [1,2,3] )